### PR TITLE
Fixed deprecation of importing collections ABC from just collections.

### PR DESCRIPTION
**Pull request by KOLANICH**

### CODE REVIEW

File: pout/compat.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
     unicode = unicode
     range = xrange # range is now always an iterator
 
-    from collections import Callable, Iterable, Set
+    from collections.abc import Callable, Iterable, Set
```
Comment (Jaymon) [inline pout/compat.py]: This line is gated with an `is_py2` check, and in Python 2 the [correct way to import the ABC classses is from the base collections module](https://docs.python.org/2/library/collections.html#collections-abstract-base-classes). You'll notice that the [is_py3 importing](https://github.com/Jaymon/pout/blob/db34e507c05e565a4ed665b985195f14e9074e7a/pout/compat.py#L53) uses the `collecitons.abc` module

File: pout/value.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 import sys
 import os
 import traceback
-from collections import KeysView
+from collections.abc import KeysView
```
Comment (Jaymon) [inline pout/value.py]: Nice catch, this should actually be deleted here and the import should be moved into the `compat.py` module under both the python 2 import and the python 3 import from the collections module

File: pout/compat.py (hunk 1 above)
Comment (KOLANICH) [reply pout/compat.py]: oops. gonna fix.

**Comment by Jaymon** [comment]
I'll merge this in when it's updated.

Thanks for submitting this and catching the error :)

**Comment by Jaymon** [comment]
I resolved this in 9faac4453452a6441240c8d7379a8ca44b242c84 so I'm closing this

**Pull request closed without merging**